### ISSUE TITLE
Map drifts away from focalPoint while zooming

### ISSUE THREAD

**Issue by fdermishin-radargeo** [mapbox/mapbox-gl-native#11642]
**Platform: Android**
**Mapbox SDK version: 5.5.0, 5.5.1**

### Steps to trigger behavior

 1. Set focal point to the center of the screen
```
val viewCenter = PointF(view.width / 2f, view.height / 2f)
map.uiSettings.focalPoint = viewCenter 
```
2. Disable scrolling, tilting and rotation of the map
```
map.uiSettings.isScrollGesturesEnabled = false
map.uiSettings.isRotateGesturesEnabled = false
map.uiSettings.isTiltGesturesEnabled = false
```
3. Zoom into the map with two fingers

### Expected behavior
focalPoint doesn't move

### Actual behavior
At first focalPoint doesn't move. However after I release my fingers, zoom continues by inertia and focalPoint moves. It seems like zooming by inertia doesn't use center of the screen, but uses center of the gesture instead.

This also happens when I don't release fingers, but zoom the map quickly.

**Comment by LukasPaczos** [comment]
Thanks for reaching out @fdermishin-radargeo.
Fix for this issue will be included in the `5.5.2`.

I also wanted to note, that we have completely reworked gestures behavior for the upcoming `6.0.0` release, so please feel free to try out `6.0.0-beta.6` or the major release itself when it's out.

**Comment by LukasPaczos** [comment]
Landed with https://github.com/mapbox/mapbox-gl-native/pull/11643.
